specification/resources/resource-manager/Microsoft.Resources/stable/2020-01-01/subscriptions.json

```diff
@@ -189,7 +189,7 @@
           "Subscriptions"
         ],
         "operationId": "Subscriptions_CheckZonePeers",
-        "description": "Compares a subscriptions logical zone mapping",
+        "description": "Compares a subscriptions logical zone mapping. Please register the feature with AZ CLI: 'az feature register -n AvailabilityZonePeering --namespace Microsoft.Resources' , and check the status with: 'az feature show -n AvailabilityZonePeering --namespace Microsoft.Resources' before calling the checkZonePeers API.",
```

Comment:
wondering if we should use some documentation link, for non-AzCLI users. say https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/preview-features?tabs=azure-portal#register-preview-feature

Reply:
@tjegbejimba could you help reivew?